versiondb/tsrocksdb/opts.go

```diff
@@ -33,20 +34,21 @@ func NewVersionDBOpts(sstFileWriter bool) *grocksdb.Options {
 	// improve sst file creation speed: compaction or sst file writer.
 	opts.SetCompressionOptionsParallelThreads(4)
 
-	// compression options at bottommost level
-	opts.SetBottommostCompression(grocksdb.ZSTDCompression)
-	compressOpts := grocksdb.NewDefaultCompressionOptions()
 	if !sstFileWriter {
+		// compression options at bottommost level
+		opts.SetBottommostCompression(grocksdb.ZSTDCompression)
```

Comment:
Use default compression(LZ4) with sst file writer can reduce the time from 21minutes to 12minutes on testnet, db size increase from 5.8G to 8.2G though, seems a good deal, the heavier compression will be gradually applied in future background compaction.